client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -260,10 +290,16 @@ const connectComponent = connect( ( state, ownProps ) => {
 		}
 	}
 
+	const isPodcastingEnabled = !! (
+		podcastingCategoryId ||
+		( podcastingCategorySlug && podcastingCategorySlug !== '0' )
+	);
+
```

Comment:
This will change a bit later when we get rid of the slug setting, but it's fine for now.